monitoring/templates/prometheus.yml

```diff
@@ -0,0 +1,47 @@
+# my global config
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+  evaluation_interval: 15s # By default, scrape targets every 15 seconds.
+  # scrape_timeout is set to the global default (10s).
+
+  # Attach these labels to any time series or alerts when communicating with
+  # external systems (federation, remote storage, Alertmanager).
+  external_labels:
+      monitor: 'ecs'
+
+# Load and evaluate rules in this file every 'evaluation_interval' seconds.
+rule_files:
+  - 'alert.rules'
+
+# alert
+alerting:
+  alertmanagers:
+  - scheme: http
+    static_configs:
+    - targets:
+      - "alertmanager:9093"
+
+scrape_configs:
+  - job_name: 'prometheus'
+    scrape_interval: 15s
+
+    static_configs:
+      - targets: ['localhost:9090']
+
+  - job_name: 'node-exporter'
+    ec2_sd_configs:
+      - region: eu-west-1
+        port: 9100
+        filters:
+          - name: "tag:Environment"
+            values: ["production"]
+          - name: "tag:Type"
+            values: ["ecs-cluster"]
+  - job_name: 'concourse'
+    scrape_interval: 15s
+    static_configs:
+      - targets: ['ci.skyscrape.rs:9391']
+  - job_name: 'cloudwatch'
+    scrape_interval: 1m
+    static_configs:
+      - targets: ['cloudwatch_exporter:9106']
```

Comment:
I would add the option to add additional targets via variables

Reply:
done